r/tests/testthat/test-dplyr-mutate.R

```diff
@@ -32,59 +47,23 @@ test_that("mutate() is lazy", {
   )
 })
 
-test_that("basic mutate", {
-  expect_dplyr_equal(
-    input %>%
-      select(int, chr) %>%
-      filter(int > 5) %>%
-      mutate(int = int + 6L) %>%
-      collect(),
-    tbl
-  )
-})
-
-test_that("mutate() with NULL inputs", {
-  expect_dplyr_equal(
-    input %>%
-      mutate(int = NULL) %>%
-      collect(),
-    tbl
-  )
-})
+# similar to https://github.com/tidyverse/dplyr/blob/master/tests/testthat/test-mutate.r#L1-L10
```

Comment:
These "similar to" comments are helpful, and I think it's worth keeping them in, but:
- At least one of them points to the wrong line numbers, so please check them and fix
- They will no longer point to the correct line numbers if `test-mutate.r` in dplyr changes, so please change the URLs to use a commit hash in place of `master`. For example, this URL could change to:
`https://github.com/tidyverse/dplyr/blob/04454209ea069939d3335c43846c85c725547a89/tests/testthat/test-mutate.r#L1-L10`. That URL will always point to the correct test even if `test-mutate.r` changes.

Reply:
100% right, last week those matched

Reply:
I don't think you addressed the second point Ian made here. You can get a permanent link like Ian mentions and use the current commit hash in the URL. One easy way to do this is to use "copy permalink" on the three dots menu when you click next to the line number in the github UI

Reply:
@pachamaltese It looks like you fixed all the other URLs except this particular one